dao/postgres/entities/pg-applications.js

```diff
@@ -120,6 +120,11 @@ class PgApplications {
                 confidential: !!appCreateInfo.confidential,
                 mainUrl: appCreateInfo.mainUrl
             };
+
+            if(appCreateInfo.description){
+              newApp.description = appCreateInfo.description.substring(0, 128);
```

Comment:
Please create a const on top of this file

`const APP_MAX_LENGTH_DESCRIPTION = 1024;`

or similar